.bit/responses/3.2-Week 3 Step 2.md

```diff
@@ -1,80 +1,181 @@
+### ✅  Task:
 
-*Note: do **NOT** use the first trigger function you coded in Week 1 – make sure you are working in the **Twilio trigger function!** Both should be in the same resource group, but they are **different** trigger functions!*
+- [ ]  Make sure you're on the `week2` branch
+- [ ]  Receive an image from a POST request using parse-multipart
+- [ ]  Upload an image using @azure/blob SDK by naming the image `test.png`
+- [ ]  Commit your updated function code to `bunnimage/index.js`
+
+### Creating a Function App
+
+We're going to have a lot of triggers in this project, so let's get started by [creating a Function App](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-azure-function)! Follow those steps to create the Function App, and then create the first HTTP trigger (this will upload our image).
+
+> **Note**: it will be helpful if you keep track of these strings for reference later in the project: Storage account name (found in "Hosting"), Function app name, Resource Group
+
+Before we start coding the trigger, though, we need to install the following `npm` packages/libraries:
+
+1. `parse-multipart`
+2. `node-fetch`
+3. `@azure/storage-blob`
+
+> **Tip**: The Azure Storage Blob client library is going to be a key piece of the project. After all, it's about blobs!
 
 <details>
-<summary>:question: Where is the documentation again?</summary>
-  </br>
-  
-The documentation is here: https://developers.giphy.com/docs/api/endpoint#translate. This link has everything you need to call the **Giphy API.**
+<summary>:question: How do I install `npm` packages?</summary>
+</br>
+
+Click on the "Console" tab in the left panel under "Development Tools".
+
+![https://user-images.githubusercontent.com/69332964/99189070-59e31d00-272d-11eb-80a4-17444e5fac65.png](https://user-images.githubusercontent.com/69332964/99189070-59e31d00-272d-11eb-80a4-17444e5fac65.png)
+
+Inside the console (shown on the right panel), type in the following commands:
+
+`npm init -y` <br />
+[`npm install parse-multipart`](https://www.npmjs.com/package/parse-multipart) <br />
+[`npm install node-fetch`](https://www.npmjs.com/package/node-fetch) <br />
+[`npm install @azure/storage-blob`](https://www.npmjs.com/package/@azure/storage-blob) <br />
 
-  <br><br/>
 </details>
 
-## **:pencil: Task 2: Call the Giphy API using the translate endpoint.**
+### Writing our *First* Azure Function to Upload an Image
 
-_Note: **Node-fetch** must be installed for the code in this step to work! It should already be installed, but you need to require it in the first lines of code._
+⬇ **Some housekeeping...**
 
-Once you are done, the function should call the Giphy API and **return the data** for a gif that matches the dominant emotion from your emotion data.
+- For the function to work, we have to initialize the packages/libraries we installed in the beginning of part 1.

```

Comment:
Modify these bullet points to match the curriculum (they currently reference content from my blog)